rmw_fastrtps_shared_cpp/src/rmw_take.cpp

```diff
@@ -60,10 +60,10 @@ _take(
   (void) allocation;
   *taken = false;
 
-  if (subscription->implementation_identifier != identifier) {
-    RMW_SET_ERROR_MSG("publisher handle not from this implementation");
-    return RMW_RET_ERROR;
-  }
+  RMW_CHECK_TYPE_IDENTIFIERS_MATCH(
+    subscription handle,
```

Comment:
Shouldn't this be something like this? that space in the macro seems odd to me. And if this is actually a problem, it is odd that CI passes
```suggestion
    subscription_handle,
```

Reply:
One of the rmw implementation files had that way of using it, and its the only time I've seen that macro being used, so I took it as an example.

Reply:
It's unusual, but since the first argument gets stringified it works. I personally don't like it, but I've seen it's used.